recipes/yahs/meta.yaml

```diff
@@ -19,11 +19,14 @@ requirements:
   host:
     - make
     - zlib
+  run:
+    - juicer
```

Comment:
Oh, by `juicer`, I do not mean the juicer_tools from the Aiden lab. It is a tool that comes with `yahs` and `agp_to_fasta` (https://github.com/c-zhou/yahs#other-tools). We might need to delete these two lines? Apologies for the misleading.